### PR TITLE
ccloudvm: update kata install URL

### PR DESCRIPTION
**Pull request by marcov**
Update the kata install commands to point to the correct repository URL.

Fixes: #326

Signed-off-by: Marco Vedovati <mvedovati@suse.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
@jodh-intel @devimc  ping

File: ccloudvm/kata-docker-xenial.yaml [hunk 1]
```diff
@@ -30,8 +30,9 @@ write_files:
 runcmd:
 
  - {{beginTask . "Install kata"}}
- - sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/release/xUbuntu_$(lsb_release -rs)/ /' >> /etc/apt/sources.list.d/kata-containers.list"
- - {{proxyVars .}} curl -sL http://download.opensuse.org/repositories/home:/katacontainers:/release/xUbuntu_$(lsb_release -rs)/Release.key | sudo apt-key add -
+ - ARCH=$(arch)
```
Comment (jodh-intel) [inline ccloudvm/kata-docker-xenial.yaml]: Nit: Safe for Ubuntu but I wonder if we want to use `uname -m` for general consistency now (see for example https://github.com/kata-containers/osbuilder/pull/227).

**Comment by jcvenegas** [comment]
Ignoring  pullapprove and merge

**Pull request merged**